scripts/sct_run_batch.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+import multiprocessing
+import subprocess
+import re
+import time
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Subjects '
+                'should be organized as folders within a single directory. We recommend '
+                'following the BIDS convention (https://bids.neuroimaging.io/). '
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables and the arguments '
+                'passed via `--task-args`',
+  formatter_class = SmartFormatter,
+  prog=os.path.basename(__file__).strip('.py'))
+
+parser.add_argument("--jobs", type = int, default = 1,
+                    help = 'The number of jobs to run in parallel. '
+                           'either an integer greater than or equal to one '
+                           'specifying the number of cores, 0 or a negative integer '
+                           'specifying number of cores minus that number. For example '
+                           '--jobs -1 indicates run ncores - 1 jobs in parallel.')
+parser.add_argument("--path-data", help = 'Setting for environment variable: PATH_DATA\n'
+                                          'Path containing subject directories in a consistent format')
+parser.add_argument("--subject-prefix", default = "sub-",
+                    help = 'Subject prefix, defaults to "sub-"')
```

Comment:
I anticipate a lot of users will not have their data organized as BIDS, so this won't work by default. How about setting the default to ''? If data *are* BIDS, the default '' will also work.

Reply:
I'll clarify in the help that this default works for BIDS, and that an empty string can be used if the subject directories don't share a prefix.

Reply:
@jcohenadad, let me know if you're happy with the new wording here.

Reply:
perfect! just a [small typo](https://github.com/neuropoly/spinalcordtoolbox/pull/2658/files#r413449751)